### PR TITLE
test(k8s): skip flush and reshard test due to operator's bug

### PR DESCRIPTION
**Pull request by vponomaryov**
The 'test_nodetool_flush_and_reshard' test fails due to the following bug: 
https://github.com/scylladb/scylla-operator/issues/1077 
So, skip it.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by vponomaryov** [comment]
The appropriate nemesis is already skipped due to this bug: https://github.com/scylladb/scylla-cluster-tests/pull/5375

File: functional_tests/scylla_operator/test_functional.py [hunk 1]
```diff
@@ -693,6 +693,7 @@ def test_default_dns_policy(db_cluster: ScyllaPodCluster):
         f"Pods: {yaml.safe_dump(pods_with_wrong_dns_policy, indent=2)}")
 
 
+@pytest.mark.skip("Disabled due to the https://github.com/scylladb/scylla-operator/issues/1077")
```
Comment (fruch) [inline functional_tests/scylla_operator/test_functional.py:696]: it still annoys me a bit we'll loss the viability of this one on EKS weekly run

I wish we could use something like:
https://docs.pytest.org/en/7.1.x/reference/reference.html#_pytest.hookspec.pytest_markeval_namespace

so we could use them inside `pytest.mark.skipIf`

but it's limit to any information we can get before any fixture is being called.

Comment (vponomaryov) [reply functional_tests/scylla_operator/test_functional.py]: Done

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**